docs/design-proposals/apiserver/Action-API-design.md

```diff
@@ -94,10 +94,42 @@ specify a desired action on Pod object. This is the recommended way for user
 have the infrastructure create Action object for a specific action for a (VM)
 Pod.
 
+Having pods/action subresource allows user/ECS to easily create actions on a
+specific Pod as illustrated in the below reboot example:
+```
+root@fw0000358:~/ActAlk# cat ../YML/reboot.json
+{
+  "apiVersion": "v1",
+  "kind": "CustomAction",
+  "actionName": "Reboot",
+  "rebootParams": {
+    "delayInSeconds": 10
+  }
+}
+
+root@fw0000358:~/ActAlk# curl -X POST http://127.0.0.1:8001/api/v1/namespaces/default/pods/1pod/action -H "Content-Type: application/json" -d @../YML/reboot.json
+```
+
+Similarly, user/ECS can request snapshot of (VM) Pod as follows:
+```
+root@fw0000358:~/ActAlk# cat ../YML/snapshot.json
+{
+  "apiVersion": "v1",
+  "kind": "CustomAction",
+  "actionName": "Snapshot",
+  "snapshotParams": {
+    "snapshotLocation": "/var/tmp/"
+  }
+}
+
+root@fw0000358:~/ActAlk# curl -X POST http://127.0.0.1:8001/api/v1/namespaces/default/pods/1pod/action -H "Content-Type: application/json" -d @../YML/snapshot.json
```

Comment:
same here.

Reply:
this is illustration of action subresource.